packages/tc-schema-sdk/lib/raw-data-wrapper.js

```diff
@@ -39,7 +87,8 @@ If npm linking the schema locally, set \`updateMode: 'dev'\`
 
 	getEnums() {
 		this.checkDataExists();
-		return this.rawData.schema.enums;
+		// TODO - allow test enum values
```

Comment:
Do test enums really impact the prod env? Feels like we could just let them be deployed without triggering any issues.

Reply:
say we add a new value to the lifecycleStage enum 'Planning' - it'd mean that would show up in both test and prod. I don't think it's a major issue, hence TODO